sms/verify-signed-sms/.env.dist

```diff
@@ -1,7 +1,7 @@
 FLASK_APP = "app.py"
 FLASK_ENV = "development"
 FLASK_RUN_PORT = 3000
-FLASK_RUN_HOST = "0.0.0.0"
+# FLASK_RUN_HOST = "0.0.0.0" Uncomment to make the server publicly available
```

Comment:
If the entire line is uncommented, will the comment itself also not be uncommented and raise a `SyntaxError`? Perhaps add a second `#` before the comment itself?

```suggestion
# FLASK_RUN_HOST = "0.0.0.0" # Uncomment to make the server publicly available
```

Reply:
Although that could also be confusing, I guess..

Reply:
I've removed this entirely now, if you run with Ngrok as we recommend it's not needed.